### ISSUE TITLE
Add a alert dialog box

### ISSUE THREAD

**Issue by MuhammedKpln** [akveo/react-native-ui-kitten#552]
Could be useful to have it. Could be a "modernish" component, not like "bootstrap" one.

**Comment by artyorsh** [comment]
Thanks for the proposal @MuhammedKpln 
We would discuss it with a team and hopefully, add into the next release plan

**Comment by shubhamkakkar** [comment]
Hey I would like to take this issue!!
Is there any design file available or is anyone working on this?

**Comment by 32penkin** [comment]
Hello @shubhamkakkar ! Thank you for your interest. Unfortunately, at the moment we do not have a design for this component so stay tuned :)

**Comment by geocfu** [comment]
I dont think that it falls into the category "nice to have". Rather to the "necessary" one.

Thank you for the awesome ui.

**Comment by CostachescuCristinel** [comment]
Hi, is there any progress on this?

BTW, suggestion for implementation: use a Modal -like component to wrap and any content and prevent presses to go trough to the back UI. Then use a Card -like component to provide the alert UI, but disabling the pressable background of the card element (as to only make pressable any buttons in Card's footer). The Card component already provides many features that an "Alert" component would need, such as header, content, footer.

**Comment by davidjuhyung** [comment]
+ 1

Alert dialogs are essential for any app, as it's often the practice to ask for confirmation "Do you really want to delete this" before the actual deletion happens.
